OpenEdXMobile/res/menu/delete_contextual_menu.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <item
+        android:id="@+id/item_delete"
+        android:icon="@drawable/ic_menu_delete"
+        android:title="@string/label_delete"
```

Comment:
The title is given for 2 reasons:

1. When user long presses the icon they can see a toast of what the icon is for.
2. Accessibility. Talkback will read out the title when the item is in focus.